### ISSUE TITLE
[Bug] [Config UI] invalid cronConfig: expected exactly 5 fields, found 1: [manual]

### ISSUE THREAD

**Issue by long2ice** [apache/incubator-devlake#2546]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

When I create a blueprint and set it to manual type, the error show: `invalid cronConfig: expected exactly 5 fields, found 1: [manual]`
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/13377178/179893212-d5edfa31-8687-4e74-ad92-3907f000d6b4.png">

<img width="1375" alt="image" src="https://user-images.githubusercontent.com/13377178/179893065-93082555-39a1-4f11-8b45-0363907f5fba.png">

### What you expected to happen

The blueprint created successfully

### How to reproduce

Just create blueprint set it to manual type

### Version

main

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by warren830** [comment]
@long2ice  Hi Jinlong, which branch are you using? I just checked by main branch, it works fine

**Comment by long2ice** [comment]
The main branch
<img width="975" alt="image" src="https://user-images.githubusercontent.com/13377178/179948592-61380a48-8b48-4fd8-af99-30f45109173c.png">

**Comment by e2corporation** [comment]
@long2ice @warren830 @klesh This is a consequence of having 'manual' as a frequency mode rather than a pure cron expression. There is already a separate flag for tracking manual mode, `isManual` I believe is the key, which UI is already correctly setting it to `true` when manual is selected, however current Backend logic expects `cronConfig` to be a pure cron expression string.

This can be solved in one of 2 ways:

1) On the frontend, the `Blueprint Manager Hook` settings payload for the blueprint should be updated to check if  cronConfig's value is string 'manual' and/or if `isManual` is true to post one of the existing cron presets, daily or hourly or any valid cron expression etc so a valid cron value is stored.

```
# Example patch in Blueprint Manager Hook (Reference Only) 
`cronConfig: cronConfig === 'manual' || mode === BlueprintMode.MANUAL ? : '0 * * * *` : cronConfig
```

2) On the backend, assuming this is possible with the current design, api should add some exception to allow `manual` as a valid value.

**Comment by e2corporation** [comment]
`isManual` fix on frontend has been added in PR #2534 @long2ice @warren830 @klesh
